Create ui-c8y-1021-22-102-subtenant-list-title-of-suspend-and-activate-button-does-not-change.md (#3517)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1021-22-102-subtenant-list-title-of-suspend-and-activate-button-does-not-change.md

---
date: ""

> title: Subtenant list: suspend and activate button title now updates correctly after toggling

Reviewer: ```suggestion
title: 'Subtenant list: suspend and activate button title now updates correctly after toggling'
```

> title: 'Subtenant list: suspend and activate button title now updates correctly after toggling'

Reviewer 2: ```suggestion
title: 'Suspend and activate button title in the subtenant list now updates correctly after toggling'
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63842
version: 1021.22.102
---

> Co-authored-by: Tristan Bastian <tristan.bastian@cumulocity.com>

Reviewer: Lets use the same message as in the other change log (https://github.com/Cumulocity-IoT/c8y-docs/pull/3516)

```suggestion
In the subtenant list, the title of the suspend and activate button did not change after toggling the button, which could lead to confusion about the current state. This has been fixed so that the button title now correctly updates to reflect the action that will be performed when clicking the button, i.e. it shows "Activate" when the subtenant is currently suspended, and vice versa. This change improves the user experience and makes the subtenant list easier to use.
```

> In the subtenant list, the title of the suspend and activate button did not change after toggling the button, which could lead to confusion about the current state. This has been fixed so that the button title now correctly updates to reflect the action that will be performed when clicking the button, i.e. it shows "Activate" when the subtenant is currently suspended, and vice versa. This change improves the user experience and makes the subtenant list easier to use.

Reviewer 2: ```suggestion
In the subtenant list, the title of the suspend and activate button did not change after toggling the button, which could lead to confusion about the current state. This issue has been fixed. The button title now correctly updates to reflect the action that will be performed when clicking the button, that is, it shows **Activate** when the subtenant is currently suspended, and vice versa. 
```
